### ISSUE TITLE
SyntaxError in get_list_display

### ISSUE THREAD

**Issue by szymborski** [elton2048/wagtail-orderable#18]
Hello,

I've found SyntaxError in return statement which prevent django startup.

Last line in get_list_display method, in OrderableMixin class.

wagtailorderable/modeladmin/mixins.py
```python
    def get_list_display(self, request):
        """Add `index_order` as the first column to results"""
        list_display = super().get_list_display(request)
        return = ('index_order', *list_display)
```

**Comment by benoitvogel** [comment]
Yes, see PR #15

**Comment by elton2048** [comment]
Fixed by #15 and included in version `1.0.2`
